addons/mail/static/src/new/composer/suggestion_service.js

```diff
@@ -219,8 +219,8 @@ class SuggestionService {
                 return 1;
             }
             if (thread?.serverData?.channel) {
-                const isMember1 = thread.serverData.channel.channelMembers[0][1].includes(p1);
-                const isMember2 = thread.serverData.channel.channelMembers[0][1].includes(p2);
+                const isMember1 = thread.channel.channelMembers[0][1].includes(p1);
```

Comment:
I don't see where `channel` is defined on `thread` model actually.

Reply:
`threadService#479`

Reply:
It should be added to thread model field definition too then.

And I would personally be in favor of defining a channel model, but I think that's more controverted.

Reply:
Added in model field definitions, channel model as a follow up if we decide its the thing to do ?